scripts/superdesk-authoring/authoring.js

```diff
@@ -2601,6 +2603,13 @@
 
                 /* End: Dateline related properties */
 
+                // watch item and save every change in history in order to perform undo/redo later
+                // ONLY for editor2 (with blocks)
+                try {
+                    angular.module('superdesk.editor2');
+                    history.watch('item', scope);
```

Comment:
is this doing some recursive wathing on the item? so it would also track changes done in sidebars etc?